packages/notification-center/src/i18n/languages/ml.ts

```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const EN: ITranslationEntry = {
```

Comment:
```suggestion
export const ML: ITranslationEntry = {
```

I think this is what causes the tests to fail.